operator/params.yaml

```diff
@@ -22,6 +22,22 @@ NODE_MEM_LIMIT_MIB:
   description: "Memory limit (in MiB) for the Cassandra node pods."
   default: 4096
 
+METRICS_CPU_MC:
+  description: "CPU request (in millicores) for the Cassandra node pods."
```

Comment:
Is this for the node pods or metrics pods?

Reply:
updated description for metrics related param.